design/images/context.puml

```diff
@@ -3,13 +3,13 @@
 
 Person(user_data_input, "Data input")
 Person(user_data_request, "Data discover and request")
-Person(user_admin, "Admin")
+Person(user_admin, "Admin", "Data owners, data managers, and other ")
 
 Person_Ext(user_results, "Explore findings and projects")
 
 Lay_Right(user_admin, user_results)
 
-System(ui, "Seedcase System for Data Resource")
+System(ui, "Seedcase Instance for a Data Resource")
```

Comment:
"Instance of a Seedcase Data Resource"?

Reply:
Hmmm, thinking on it, I'm not sure either sounds right to me. I've modified it, maybe this is better. We'll have more reviews of this over time.